### ISSUE TITLE
Memory Leaks on Linux

### ISSUE THREAD

**Issue by Andrew-Lees11** [IBM-Swift/BlueRSA#37]
We had reports of Memory leaks when signing/verifying a with Swift-JWT. We recreated this on our own repo [JWT](https://github.com/Andrew-Lees11/testJWT) along with encryption and decryption. This produced the following memory report: [leaks.txt](https://github.com/IBM-Swift/BlueRSA/files/2906745/leaks.txt) which highlights leaks at:
```
evp_key = .init(PEM_read_bio_PUBKEY(bio, nil, nil, nil))
``` 
This is due to the evp_key not being properly freed after it was referenced by:
```
var pkey_ctx = EVP_PKEY_CTX_new(evp_key, nil)
```
(Mirrored for private key).

There are also leaks in encrypt and decrypt where allocated memory from pointers wasn't being deallocated.

**Comment by Andrew-Lees11** [comment]
These leaks are fixed by PR #36

**Comment by Andrew-Lees11** [comment]
Closing now fix has been merged and release in BlueRSA 1.0.24
